src/backend/Dockerfile

```diff
@@ -8,26 +8,20 @@ ENV PYTHONUNBUFFERED=1
 
 WORKDIR /app
 
-# Copy the backend source code, excluding the pretrained models (see .dockerignore)
-COPY . .
-
-RUN git submodule init && git submodule update --remote --merge
-
-# Download and unzip pretrained models
-RUN curl -L -o /app/src/backend/sinr/pretrained_models.zip https://data.caltech.edu/records/dk5g7-rhq64/files/pretrained_models.zip?download=1 && \
-    unzip -o /app/src/backend/sinr/pretrained_models.zip -d /app/src/backend/sinr && \
-    rm /app/src/backend/sinr/pretrained_models.zip
+# Download pretrained models
+RUN curl -L -o /app/pretrained_models.zip https://data.caltech.edu/records/dk5g7-rhq64/files/pretrained_models.zip?download=1
 
-# Create a new conda environment with Python 3.9
-RUN conda create -n inatator python=3.9
+COPY ./src/backend/requirements.txt .
 
 # Install Python dependencies
-RUN conda run -n inatator \
-    pip install -r ./src/backend/requirements.txt
+RUN pip install -r ./requirements.txt
 
-# Make the entrypoint script executable
-RUN chmod +x /app/src/backend/entrypoint.sh
+# Copy the backend source code, excluding the pretrained models (see .dockerignore)
+COPY . .
+
+RUN git submodule init && git submodule update --remote --merge
 
-EXPOSE 8000
+# Unzip pretrained models
+RUN unzip -o /app/pretrained_models.zip -d /app/src/backend/sinr
 
-ENTRYPOINT [ "/app/src/backend/entrypoint.sh" ]
```

Comment:
Can you remove entrypoint.sh from repo if we are not using it

Reply:
done in new commit